Anlab.Mvc/Views/Admin/MailQueue.cshtml

```diff
@@ -0,0 +1,125 @@
+
+        </tr>
+    </thead>
+    <tbody style="vertical-align: top;">
+    @foreach (var message in Model)
+    {
+        <tr>
+            <td>
+
+                @if(message.Sent.HasValue && message.Sent == true)
+                {
+                    <p>Yes</p>
+                }
+                else
+                {
+                    <p>No</p>
+                }
+            </td>
+            <td>@message.SendTo
+                
+            </td>
+            <td data-search="@message.Subject">
+                @message.Subject<br />
+            <a data-toggle="collapse" href="#@message.Id" aria-expanded="true" aria-controls="@message.Id">
+                Expand Body
+            </a> |
+            <a href="ViewMessage/@message.Id" target="_blank">Open in new window</a>
+            <div id="@message.Id" class="panel-collapse collapse">
+                <div class="panel-body">
+                    @Html.Raw(message.Body)
+                    </div>
+                    <a data-toggle="collapse" href="#@message.Id" aria-expanded="true" aria-controls="@message.Id">
+                        Collapse Body
+                    </a> |
+                    <a href="ViewMessage/@message.Id" target="_blank">Open in new window</a>
+                
+            </div>
+        </td>
+            <td>
+
+            @if (message.Sent.HasValue && message.Sent==true)
+            {
+                <p> 
+                Created At: @message.CreatedAt.ToLocalTime()
+                <br />
+                Sent At: @message.SentAt.ToLocalTime()</p>
+            }
+            else
+            {
+                <p>
+                Created At: @message.CreatedAt.ToLocalTime()
+                <br />
+                Error: @message.FailureReason </p>
+            }
+            </td>
+
+        </tr>
+    }
+
+    </tbody>
+</table>
+    </div>
+
+@section AdditionalStyles
+{
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css" />
+    <style>
+        div.panel-body tr tr{
+            background-color: transparent;
+        }
+    </style>
+
+}
+@section Scripts
+{
+    <script type="text/javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
+    <script type="text/javascript">
+        $(function() {
+            $("#table").dataTable({
+                "order": [[0, "asc"],[3,"desc"]],
+                "columnDefs": [
+                    {
+                        "type": "ordering", "targets": [0, 1, 2, 3]
+
+                    },
+                    {
+                        "width": "5%", "targets": [0]
+
+                    },
+                    {
+                        width: "55%", "targets":[2]
+
+                    }
+                ]
+            });
+            $("#collapsebutton").hide();
+            $("#expandbutton").click(function () {
+                $('div.panel-collapse').addClass('in').css("height", "");
```

Comment:
I think the collapse function supports a .collapse() call.

http://getbootstrap.com/javascript/#via-javascript-3